appcues/src/main/java/com/appcues/data/mapper/experience/ExperienceMapper.kt

```diff
@@ -95,18 +96,22 @@ internal class ExperienceMapper(
             experiment = experiments?.getExperiment(from.id),
             completionActions = arrayListOf<ExperienceAction>().apply {
                 from.redirectUrl?.let {
-                    add(LinkAction(
-                        redirectUrl = it,
-                        linkOpener = scope.get()
-                    ))
+                    add(
+                        LinkAction(
+                            redirectUrl = it,
+                            linkOpener = scope.get()
+                        )
+                    )
```

Comment:
Looks like this will have some sort of conflict with https://github.com/appcues/appcues-android-sdk/pull/391 perhaps, depending which goes first.